tools/webpack.config.ts

```diff
@@ -134,7 +134,9 @@ export default (env?: string) : webpack.Configuration => {
         { path: "../node_modules/underscore/underscore.js", name: '_' },
         { path: "../node_modules/handlebars/dist/handlebars.runtime.js", name: 'Handlebars' },
         { path: "../node_modules/to-markdown/dist/to-markdown.js", name: 'toMarkdown' },
-        { path: "../node_modules/sortablejs/Sortable.js"}
+        { path: "../node_modules/sortablejs/Sortable.js"},
+        { path: "../node_modules/plotly.js/dist/plotly-basic.min.js", name: 'Plotly'},
```

Comment:
plotly doesn't need to be in the app bundle; it's used from `stats.js` which is for the independent `/stats` page.